package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "node scripts/install-web.js && npx concurrently npm:build:watch npm:dev:server npm:dev:client",
     "dev:client": "cd examples && cd react-web && npm start",
     "dev:server": "cross-env NODE_ENV=development nodemon -w src -w examples -e js,ts --exec babel-node --extensions '.ts,.js' --ignore 'src/**/*.test.ts' --presets @babel/preset-env examples/react-web/server.js",
-    "build": "cross-env BABEL_ENV=rollup rollup --config rollup.config.js",
+    "build": "cross-env BABEL_ENV=rollup rollup --config rollup.config.js --silent",
```

Comment:
How are we going to catch any actual warnings that Rollup throws if they are silenced?

Reply:
Actual warnings will  still be on stdout. Just not the build info of what got build (which weirdly is also on stderr).